.travis.yml

```diff
@@ -1,25 +1,27 @@
 sudo: required
-dist: xenial
+
+dist: trusty
+
+addons:
+  apt:
+    sources:
+    - ubuntu-toolchain-r-test
+    packages:
+    - gcc-4.9
+    - g++-4.9
+    - wget
 
 branches:
   except:
   - stable
 
-lang: go
-
-go:
-  - 1.7.x
+language: cpp
 
 jdk:
   - oraclejdk8
```

Comment:
These changes are not part of rate limiting.   Could you merge master to your branch first to get rid of these changes?